ImageD11/frelon_peaksearch.py

```diff
@@ -423,7 +423,7 @@ def process_dataset_for_peaks_columnfile(
     all_frames_peaks_list = collect_all_frames_peaks(
         dataset.masterfile,
         "%s/measurement/%s"%(dataset.scans[scan_number], dataset.detector),
-        dataset.omega[0,:],
+        dataset.omega[scan_number,:],
```

Comment:
Please delete the  `,:` and have `dataset.omega[scan_number]` so that we can have a list of omega in dataset (in the future)